src/Qir/Runtime/lib/QIR/README.md

```diff
@@ -0,0 +1,61 @@
+# API Dependency
+
+(See the raw file. Please keep the raw file readable rather than the browser-rendered one)
```

Comment:
I experimented with different content, including tables. I ended up in some balance between the [raw file](https://raw.githubusercontent.com/microsoft/qsharp-runtime/34040a7abab04ad041ec897bf2399397a93dc674/src/Qir/Runtime/lib/QIR/README.md) readability and the [web view](https://github.com/microsoft/qsharp-runtime/blob/34040a7abab04ad041ec897bf2399397a93dc674/src/Qir/Runtime/lib/QIR/README.md) readability. 
Since the file is intended to be mostly used _during coding_, I wanted the raw file readability to be higher priority.

Rewording this fragment to:
> (Try to keep the readability balance between the web view and raw file, give the preference to the raw file)